docs/2.7.0/docs/canton/usermanual/security.rst

```diff
@@ -476,14 +507,56 @@ You can optionally pass a wrapper key id to change to or let Canton generate a n
 KMS configuration. If you wish to change the key specification (e.g. enable multi region) you are required
 to update the configuration before rotating the wrapper key.
 
+Canton Configuration for External Key Storage and Usage
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. warning::
+    You cannot mix an external private key storage configuration
+    with an encrypted private key storage configuration.
+
+In the example below we configure a Canton participant node (called ``participant1``) to generate and
+store private keys in an external KMS. Besides the previously presented :ref:`AWS KMS configuration <kms_config>`
+you only need to specify the correct crypto provider ``kms``:
+
+.. literalinclude:: /canton/includes/mirrored/enterprise/app/src/test/resources/encrypted-store-enabled-tagged.conf
+   :language: none
+   :start-after: user-manual-entry-begin: PrivateKeyStoreConfig
+   :end-before: user-manual-entry-end: PrivateKeyStoreConfig
+
+Please be aware that AWS KMS only supports the following cryptographic schemes:
+
+.. _kms_schemes:
+
+Signing:
+
+- ECDSA with P-256 and P-384
+
+Asymmetric Encryption:
+
+- RSA 2048 with OAEP using SHA-256
+
+Therefore, a node running with a ``kms`` provider is only ever able to communicate
+with other nodes running a ``kms`` or ``jce`` providers.
+
+An example configuration that puts it all together is below:
+
+.. literalinclude:: /canton/includes/mirrored/enterprise/app/src/test/resources/encrypted-store-enabled.conf
+   :language: none
+
+.. warning::
+    Once a node starts with a KMS as its provider it can no longer be reverted without a full reset of the node
+    (i.e., database pruning and re-generation of all keys).
+
+.. _manual-aws-ksm-key-rotation:
+
 Auditability
 ^^^^^^^^^^^^
 
 AWS provides  tools to monitor KMS keys. To set automatic external logging, refer to the `official documentation
 <https://docs.aws.amazon.com/kms/latest/developerguide/monitoring-overview.html>`_.
 This includes instructions on how to set AWS Cloud Trail or Cloud Watch Alarms
-to keep track of usage of KMS keys. Errors resulting from the use of the
-wrapper key (i.e., during encryption and decryption) are logged in Canton.
+to keep track of usage of KMS keys. Errors resulting from the use of a
```

Comment:
we also need to extend that with an optional audit trail for all the KMS usage when used in full KMS mode

Reply:
_...AWS Cloud Trail or Cloud Watch Alarms to keep track of usage of KMS keys or of the KMS operations in general._